cmd/kube-version-change/version.go

```diff
@@ -39,7 +39,7 @@ var (
 	inputSource   = flag.StringP("input", "i", "-", "Input source; '-' means stdin")
 	outputDest    = flag.StringP("output", "o", "-", "Output destination; '-' means stdout")
 	rewrite       = flag.StringP("rewrite", "r", "", "If nonempty, use this as both input and output.")
-	outputVersion = flag.StringP("out-version", "v", latest.GroupOrDie("").Version, "Version to convert input to")
+	outputVersion = flag.StringP("out-version", "v", latest.GroupOrDie("").GroupVersion.Version, "Version to convert input to")
```

Comment:
I don't think it's useful to support multiple group/versions in the outputVersion flag of the conversion tool. IMO, if a user needs to convert multiple objects belong to different groups, it's clearer if he calls the tool multiple times.